servers/audio/audio_server.cpp

```diff
@@ -1839,9 +1842,75 @@ String AudioServer::get_input_device() {
 }
 
 void AudioServer::set_input_device(const String &p_name) {
+	// how about: if (!p_is_active) here?
 	AudioDriver::get_singleton()->set_input_device(p_name);
 }
 
+Error AudioServer::set_input_device_active(bool p_is_active) {
+	if (input_device_active == p_is_active) {
+		return OK;
+	}
+	if (p_is_active) {
```

Comment:
Nitpick, but I would use `if/else` instead of early return here, because `p_is_active` and `!p_is_active` are of equal importance, and neither is an 'early return case' of the other.
If you do want to use early return syntax, I'd probably put the shorter one in the early return.